src/Pim/Product/CreateProductCommand.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * @copyright Copyright (c) Wizacha
+ * @license Proprietary
+ */
+declare(strict_types=1);
+
+namespace Wizaplace\SDK\Pim\Product;
+
+final class CreateProductCommand extends ProductUpsertData
```

Comment:
Vous allez me dire : 

> Mais Hector, pourquoi as tu créé une classe abstraite qui n'est étendue que par une classe vide ?!?!?

et c'est une très bonne question, merci de la poser.

L'idée est que le `ProductUpsertData` va être réutilisé pour une `UpdateProductCommand`, qui aura un champ supplémentaire : l'id du produit.

> Mais pourquoi pas avoir  `UpdateProductCommand extends CreateProductCommand` dans ce cas ?

Encore une bonne question, you're on fire :fire: ! Et bien c'est pour être futur-proof : si on veut rajouter un champ dans la commande de création mais pas dans la commande d'update, cette structure rend ca possible.